mitodl/open-discussions
# Split profiles into chunks for indexing

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #1434 

#### What's this PR do?
- Modifies the `index_profiles` task to run on a list of profile_ids
- Modifies the `start_recreate_index` task to split the list of profile id's into chunks (based on the value of `ELASTICSEARCH_INDEXING_CHUNK_SIZE`), which are then sent to the `index_profiles` task.
- Filters out the indexing user
- Adds `ELASTICSEARCH_INDEXING_CHUNK_SIZE` to `app.json` and `.env.example` (it is already used elsewhere in the app).

#### How should this be manually tested?
- Create lots of new users and profiles
- Run `docker-compose run web python manage.py recreate_index`
- Ensure that the task ends successfully and all profiles are present in the `discussions_local_profile_default` index.
